server/sonar-server/src/main/java/org/sonar/server/search/Result.java

```diff
@@ -75,31 +72,52 @@ public Result(@Nullable BaseIndex<K, ?, ?> index, SearchResponse response) {
 
   private void processAggregation(Aggregation aggregation) {
```

Comment:
In fact your right, I thought SearchResponse could be used externally of the Result class.
Ok for an internal class, and maybe we should think of another name for this classe, because it's main purpose is to decorate facets on a search query.